src/Symfony/Component/Notifier/Bridge/KazInfoTeh/KazInfoTehTransportFactory.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\KazInfoTeh;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Egor Taranov <dev@taranovegor.com>
+ */
+final class KazInfoTehTransportFactory extends AbstractTransportFactory
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+
+        if ('kazinfoteh' !== $scheme) {
+            throw new UnsupportedSchemeException($dsn, 'kazinfoteh', $this->getSupportedSchemes());
+        }
+
+        $username = $this->getUser($dsn);
+        $password = $this->getPassword($dsn);
+        $sender = $dsn->getRequiredOption('sender');
+        $host = 'default' === $dsn->getHost() ? null : $dsn->getHost();
+        $port = $dsn->getPort();
+
+        return (new KazInfoTehTransport($username, $password, $sender, $this->client, $this->dispatcher))->setHost($host)->setPort($port);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getSupportedSchemes(): array
+    {
+        return ['kazinfoteh'];
```

Comment:
As you've kept KazInfoTeh, I would use `kaz-info-teh` as the scheme here for consistency.

Reply:
Hello

Fixed in symfony/symfony#44360, symfony/symfony-docs#16088 and symfony/recipes#1031.

Should we also replace `KAZINFOTEH_DSN` replace with `KAZ_INFO_TEH_DSN`?

Thanks.

Reply:
> Should we also replace KAZINFOTEH_DSN replace with KAZ_INFO_TEH_DSN?

Yes please